ceph.spec.in

```diff
@@ -42,11 +42,6 @@
 %endif
 %endif
 
-%if %{with selinux}
-# get selinux policy version
-%{!?_selinux_policy_version: %global _selinux_policy_version %(sed -e 's,.*selinux-policy-\\([^/]*\\)/.*,\\1,' /usr/share/selinux/devel/policyhelp 2>/dev/null || echo 0.0.0)}
```

Comment:
We can just keep this line but drop parsing the policyhelp file so we will end up with this:
`%{!?_selinux_policy_version: %global _selinux_policy_version 0.0.0}`